### PR TITLE
created generic TypeAheadSelct component,

### PR DESCRIPTION
**Pull request by ajaypratap003**
### Type of change

- Enhancement / new feature
- Refactoring

### Description
created generic TypeAheadSelct component,replace existing code with TypeAheadSelct
and removed unused varibale, cofde

### Checklist

- [ ] Update/write design documentation in `./documentation/design`
- [ ] Write tests and make sure they pass
- [ ] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

**Comment by anukritijha** [comment]
LGTM

File: console/console-init/ui/src/modules/address-detail/components/AddressLinksToolbar/AddressLinksToolbar.tsx [hunk 1]
```diff
@@ -1,56 +1,36 @@
...
-    chip: string | DataToolbarChip
-  ) => void;
+export interface IAddressLinksToolbarProps
+  extends IAddressLinksToggleGroupProps {
   sortValue?: ISortBy;
   setSortValue: (value: ISortBy) => void;
   onClearAllFilters: () => void;
 }
-const AddressLinksToolbar: React.FunctionComponent<IAddressLinksToolbarProps> = ({
+const AddressLinksToolbar: React.FunctionComponent<IAddressLinksToolbarProps &
```
Comment (drcoolsanjeev) [inline console/console-init/ui/src/modules/address-detail/components/AddressLinksToolbar/AddressLinksToolbar.tsx:26]: why there are two types? I thinks we already extended the second interface DataToolbarContentProps in the IAddressLinksToolbarProps.

File: console/console-init/ui/src/modules/address-space/components/MessagingToolbar/MessagingToolbar.tsx [hunk 2]
```diff
@@ -1,53 +1,34 @@
+/*
+ * Copyright 2020, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
 import React from "react";
-import { MessagingToolbarToggleGroup } from "../MessagingDatatoolbarToggleGroup";
+import {
```
Comment (drcoolsanjeev) [inline console/console-init/ui/src/modules/address-space/components/MessagingToolbar/MessagingToolbar.tsx:7]: I think we can update this path.

File: console/console-init/ui/src/modules/address-detail/components/AddressLinksToolbar/AddressLinksToolbar.tsx (hunk 1 above)
Comment (ajaypratap003) [reply console/console-init/ui/src/modules/address-detail/components/AddressLinksToolbar/AddressLinksToolbar.tsx:26]: second one interface is the part of PatternFly. as we are using PF component in this file.In previous code this interface extends with component interface but sometimes 2 interfaces cause the issue due to incompatible types. might be we can pass some properties as same name as available in PF interface. that's why we have both interface.

File: console/console-init/ui/src/modules/address-space/components/MessagingToolbar/MessagingToolbar.tsx (hunk 2 above)
Comment (ajaypratap003) [reply console/console-init/ui/src/modules/address-space/components/MessagingToolbar/MessagingToolbar.tsx:7]: yes, this has fixed

**Pull request merged**